tests/e2e/support/commands.js

```diff
@@ -1095,19 +1089,19 @@ Cypress.Commands.add('concedeOpponent', (gameId = null) => {
   cy.makeSocketRequest('game', 'concede', { moveType: MoveType.CONCEDE }, 'POST', gameId);
 });
 
-Cypress.Commands.add('stalemateOpponent', () => {
+Cypress.Commands.add('stalemateOpponent', (gameId = null) => {
```

Comment:
Added `gameId` param so that we can call these from the home page and have it apply to a specific game by its id since the game id is not taken from the requesting user's session anymore